CHANGES.txt

```diff
@@ -25,6 +25,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Added -fsanitize support to ParseFlags().  This will propagate to CCFLAGS and LINKFLAGS.
 
   From Ryan Saunders:
+    - Added VCPkg() builder, enabling cross-platform acquisition and building of 3rd party C/C++
```

Comment:
Please add a blurb in release.txt as well.
Does this require having cmake installed? If so you should mention that here and check in your builder if cmake is installed.

Reply:
VCPkg does not require CMake installed. Instead, it downloads a private copy of CMake. I've updated release.txt.

Reply:
Gotcha.

Reply:
Please don't mark conversations you did not start as resolved. That is for the initial commenter to do.

Reply:
Ah, ok. Thanks for letting me know. (Azure DevOps pull requests follow a different convention (author marks "resolved" and commentor can then "close" the thread if they agree).